source/index.html.md

```diff
@@ -225,16 +242,16 @@ curl \
 ```
 `POST /users`
 
-This method is create a new user in the Betable API and should be called at the same time that a new user is created in the vendor's platform.
+This method is create a new user in the Betable API and should be called at the same time that a new user is created in the partner's platform.
 
-### Query Parameters
+### Request Parameters
```

Comment:
Query Parameters are the things after the `?` in the URL, I changed these all to Request Parameters if its OK